test/test_code_generation.py

```diff
@@ -54,19 +54,27 @@ def test_generate_weblab_model(tmp_path):
     model = os.path.join('test', 'models', 'hodgkin_huxley_squid_axon_model_1952_modified.cellml')
     model = cellmlmanip.load_model(model)
 
-    # Select model outputs
-    outputs = [
-        VariableReference.create('oxmeta', OXMETA_NS, 'membrane_fast_sodium_current'),
```

Comment:
I've not reviewed actions.py yet. Do we still need the `create` method if this code is gone?